### DISCUSSION TITLE
can't delete model in noctua form editor

### DISCUSSION THREAD

**Discussion by cerivs** [geneontology/helpdesk#347] [Q&A] [answered]
It looks like the rows in the form editor no long support deletion.  Is this intentional?  How are we supposed to incorrect annotations in the form editor?  I can only see how to delete the entire set of annotations and I only need to delete one.  Can this information be added to the noctua help pages?

Thanks,
Ceri

**Comment by vanaukenk** [comment, marked as answer]
Hi @cerivs 
You can delete annotations from the graph editor.
We've been conservative about deleting annotations from the form so that curators don't accidentally delete interconnected annotations.
I'll review the documentation to make this clearer.
If you need more help, let me know and we can look at this together.
